packages/docusaurus-theme-classic/src/theme/BlogPostItem/index.tsx

```diff
@@ -117,7 +117,13 @@ function BlogPostItem(props: Props): JSX.Element {
           <footer className="row margin-vert--lg">
             {tags.length > 0 && (
               <div className="col">
-                <strong>Tags:</strong>
+                <strong>
+                  <Translate
+                    id="theme.BlogPostItem.tags"
```

Comment:
we'll have a similar page/label for docs as docs are going to have tags soon, and feel we'll be able to factorize so the id may be updated later to "theme.TagList.label" or something.